swap_meet/item.py

```diff
@@ -0,0 +1,23 @@
+class Item:
+    def __init__(self, category = ""):
+        # default parameter is what we want in case nothing is passed
+        self.category = category
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition == 5:
```

Comment:
This function uses self.condition, but self.condition is not initialized in __init__.  The following code will throw an exception:
```
my_item = Item("Stuff")
my_item.condition_description()
```